packages/hoprd/crates/hopr-lib/src/chain.rs

```diff
@@ -71,6 +71,8 @@ pub struct ChainOptions {
     pub max_priority_fee_per_gas: String,
     pub native_token_name: String,
     pub hopr_token_name: String,
+    /// expected block time on the chain in milliseconds
+    pub block_time: u64,
```

Comment:
This is hardly relevant for non-testing chains, right? Should it be configured like this?

Reply:
It is relevant for any chain as long as we have static indexer intervals since the block time differs on every chain.